frontend/src/views/GNewShoot.vue

```diff
@@ -243,11 +244,15 @@ export default {
       }
 
       try {
-        await this.api.createShoot({
+        const { headers } = await this.api.createShoot({
           namespace: this.shootNamespace,
           data: this.shootManifest,
         })
-        this.setSuccess('Cluster created')
+        if (headers.warning) {
+          this.setPersistentWarning({ message: headers.warning })
```

Comment:
The text is not renderd correctly. Long strings are cut off. We could use https://github.com/marcbachmann/warning-header-parser or create our own implementation.

Reply:
Added parser